dashboard/src/routes/~tree/~$treeId/~index.tsx

```diff
@@ -18,6 +18,10 @@ const treeDetailsSearchSchema = z.object({
   diffFilter: zDiffFilter,
   testPath: z.string().optional().catch(''),
   origin: zOrigin,
+  gitUrl: z.string().optional().catch(''),
+  gitBranch: z.string().optional().catch(''),
+  treeName: z.string().optional().catch(''),
+  commitName: z.string().optional().catch(''),
```

Comment:
can you put this in a single object? you can find examples of zod object in the `types` folder

Reply:
maybe this object can be called `treeInfo`